_docs/Qabel-Protocol-Box.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Protocol: box"
+---
+# Box Specification
+
+## Abstract
+
+A set of protocols to store files and folders on a VOLUME that is on S3 and managed by a third party who doesn't need to be trusted.
+
+## Used services
+
+Qabel Box uses an Accounting server which controls the access to S3. Every client who needs write access has to be authenticated by the server and then receives a set of credentials that he can use for direct access to the VOLUME.
+Qabel Box also directly uses an S3 compatible server to store the blocks and metadata.
```

Comment:
Yes. I think I'll just remove the "compatible" for now, at least until we exactly know which features from AWS we'll use.